### ISSUE TITLE
Angular material + Tailwind: want to setup a clean ng project from scratch

### ISSUE THREAD

**Issue by E-jarod** [ngneat/tailwind#14]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[X] Feature request
[ ] Documentation issue or request
[X] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

There is two case: 
This one is working : 
```bash
ng new ng-material-tailwind
ng add @angular/material # ok
ng add @ngneat/tailwind # ok
```
But I have css issues : Material's styles override Tailwind's styles and `@apply` or `#{!mportant} throw errors at compilation : 
![2020-10-24-021430_413x714_scrot](https://user-images.githubusercontent.com/57091681/97063218-aad47b00-159e-11eb-8264-69c222adfe18.png)
![2020-10-24-162017_842x601_scrot](https://user-images.githubusercontent.com/57091681/97084093-d98d3880-1614-11eb-9968-aedda0190a40.png)

And this one is not working (because material need the official schematics) : 
```bash
ng new ng-tailwind-material
ng add @ngneat/tailwind # ok
ng add @angular/material # not ok
# ...
✔ Packages installed successfully.
Your project is not using the default builders for "build". The Angular Material schematics cannot add a theme to the workspace configuration if the builder has been changed
```
![2020-10-24-020910_1583x447_scrot](https://user-images.githubusercontent.com/57091681/97063077-0ce0b080-159e-11eb-8cbc-c5f6accea6dd.png)

## Expected behavior

- [ ] I wish to make tailwind override Material's styles in the first repo [here](https://github.com/E-jarod/ng-material-tailwind) (I simply don't know how to)
- [ ] `ng add @ngneat/tailwind` should works independently of `ng add @angular/material`

## What is the motivation / use case for changing the behavior?

Just a new cli project, and `ng add` command.

## Environment

<pre><code>
Angular CLI: 10.0.8
Node: 12.18.3
OS: linux x64

Angular: 10.0.14
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1000.8
@angular-devkit/build-angular     0.1000.8
@angular-devkit/build-optimizer   0.1000.8
@angular-devkit/build-webpack     0.1000.8
@angular-devkit/core              10.0.8
@angular-devkit/schematics        10.0.8
@angular/cdk                      10.2.5
@angular/cli                      10.0.8
@angular/material                 10.2.5
@ngtools/webpack                  10.0.8
@schematics/angular               10.0.8
@schematics/update                0.1000.8
rxjs                              6.5.5
typescript                        3.9.7
webpack                           4.43.0

Browser:
- [X] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: v12.18.3
- Platform:  Manjaro i3
- Others: VSCode, yarn
</code></pre>

**Comment by E-jarod** [comment]
Hello @nartc, do you find out what is the problem ?

**Comment by nartc** [comment]
@E-jarod Hi, I have not had time to dive into this issue as I've been going through some new job transitioning myself. I'll take a closer look asap. Sorry for any inconvenience.

**Comment by E-jarod** [comment]
@nartc Ok I see
By the way, I have added just yesterday this line in `tailwind.config.js`

```diff
module.exports = (isProd) => ({
+  important: true,
  // ...
```

This line overrides material's styles, as I wished.

Now I just need to know how to make the `@apply` works in `.scss` files !

And the other issue also : make `ng add @ngneat/tailwind` and `ng add @angular/material` working independently, if possible.

**Comment by nartc** [comment]
Hi @E-jarod , I'll add more documentations on various UI libraries in the near future. Sorry for the delay and thanks!

**Comment by nartc** [comment]
Closed in favor of #24
